docs/releasenotes/archive/service/year2020.md

```diff
@@ -5,4 +5,662 @@ sidebar_label: 2020 Archive
+
+Query:\
+`<code>_sourceCategory=stag/xmltest ("678" or "emptytag")\
+| limit 5\
+| parse xml "/users/user[2]" as first_name nodrop`
+
+**Old lib** would return: `<user          id="456" role="contributor"><first_name>Bob</first_name><last_name>Smith</last_name><email>bob@emailplace.com</email></user>`\
+**New lib** would return: `<user id="456" role="contributor"><first_name>Bob</first_name><last_name>Smith</last_name><email>bob@emailplace.com</email></user><code>`
+
+---
+## October 23, 2020 (Observability)
+
+Update - We have rolled out audit logging support for [Monitors](https://help.sumologic.com/Visualizations-and-Alerts/Alerts/Monitors "Monitors"). Create, read, update, and delete operations of Monitors are logged in the [Audit Event Index](https://help.sumologic.com/Release-Notes/Service-Release-Notes "Service Release Notes").
+
+---
+## October 23, 2020 (Apps)
+
+New - We're happy to announce the [Windows JSON](https://help.sumologic.com/07Sumo-Logic-Apps/04Microsoft-and-Azure/Windows_JSON "https://help.sumologic.com/07Sumo-Logic-Apps/04Microsoft-and-Azure/Windows_JSON") App based on the JSON event log format and provides insight into the operations of the Windows system operation and events so that you can better manage and maintain your environment. The Windows JSON App is based on the JSON Windows event log format and consists of predefined searches and dashboards that provide visibility into your environment for real-time analysis of overall usage of Security Status, System Activity, Updates, User Activity, and Applications.
+
+![Windows_Overview.png](https://help.sumologic.com/@api/deki/files/10238/Windows_Overview.png?revision=1&size=bestfit&width=601)
+
+New - [Active Directory JSON](https://help.sumologic.com/07Sumo-Logic-Apps/04Microsoft-and-Azure/Active_Directory_JSON) assists you in monitoring the Windows Active Directory deployment by analyzing Active Directory logs in the JSON based event log format. The app includes predefined searches and dashboards that provide user activity into your environment for real-time analysis of overall usage.
+
+![Active Directory Service Activity.png](https://help.sumologic.com/@api/deki/files/10239/Active_Directory_Service_Activity.png?revision=1&size=bestfit&width=600)
+
+---
+## October 21, 2020 (Manage)
+
+Update - [Webhook payload variables](https://help.sumologic.com/Manage/Connections-and-Integrations/Webhook-Connections/Set_Up_Webhook_Connections#Webhook_payload_variables "Set Up Webhook Connections") need to be in mustache format. You do this by wrapping each variable in double curly brackets, like `{{variable}}`. In ---## February 2018 we introduced this format. We did not deprecate the previous format using a dollar sign, like `$variable`. Going forward, we will only support mustache format.
```

Comment:
Pls fix formatting where it says "In ---## February 2018"

Reply:
Fixed.